Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -180,3 +180,29 @@ def customiseRun2CommonHI(process):
 
     return process
 
+##############################################################################
+# modifications of muon DQMOffline for pPb run
+def customiseRun2PPB_MuonDQMOffline(process):
```

Comment:
Hi @goni , this is a customization function. We can call it anywhere as long as it is in the release. Go ahead and move the function to DQMOfflineHeavyIons_cff.py please.